README.md

```diff
@@ -189,89 +300,49 @@ export function a11yAuditIf(contextSelector, axeOptions) {
 }
 ```
 
-#### Setting Options
+### Logging violations to the console
 
-You can provide options to axe-core for your tests using the `setRunOptions` API. Options can
-be set a few ways:
+This addon provides the capability of summarizing all violations found during tests, and outputting those failures to the console once the test suite is completed. To enable this functionality, import `setupConsoleLogger` and invoke in your `tests/test-helper.js` file:
```

Comment:
This says the name is `setupConsoleLogger` but the actual invocation below shows `setupConsoleReporter`, which is it?

Reply:
Oops. That was a mistake. Fixed.